CONTRIBUTING.md

```diff
@@ -281,20 +250,68 @@ sh tools/install_so_files.sh    # PowerShell
 ```
 
 Note that you need bazel, a C++ compiler and a NVCC compiler (if you want to test
-Cuda ops). For that reason, we recommend you [run inside the custom-op docker containers](#run-manually)
+Cuda ops). For that reason, we recommend you run inside the custom-op docker containers. 
+This will avoid you the hassle of installing Bazel, GCC/clang...
+See below.
 
 
-#### Testing with Pycharm
+#### Run Manually
 
-Pycharm has a debugger build in the IDE for visual inspection of variables
-and step by step executions of Python instructions. It can run your test 
-functions from the little green arrows next to it. And you can add 
- breakpoints by just clicking next to a line in the code (a red dot will appear). 
- 
- But in order for the debugger to run correctly, you need to specify 
- that you use pytest as your main test runner, not unittest (the default one). 
- 
- For that, go in File -> Settings -> search box -> Default test runner -> Select "Pytest".
+Running tests interactively in Docker gives you good flexibility and doesn't require 
+to install any additional tools.
+
+CPU Docker: 
+```
+docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:2.1.0-custom-op-ubuntu16
+```
+
+GPU Docker: 
+```
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:2.1.0-custom-op-gpu-ubuntu16
+```
+
+Configure:
+```
+python3.5 ./configure.py  # Links project with TensorFlow dependency
+```
+
+Install in editable mode
+```
+python3.5 -m pip install -e .
+python3.5 -m pip install pytest pytest-xdist
+```
+
+Compile the custom ops
+```
+bash tools/install_custom_ops.sh
+```
+
+Run selected tests:
+```bash
+python3.5 -m pytest path/to/file/or/directory/to/test
+```
+
+#### Testing with Bazel
+
+Testing with Bazel is still supported but not recommended unless you work at Google.
+This is because pytest offers many more options to run your test suite and have
+better error reports, timings reports, open-source plugins and documentation online, 
+at least for Python testing. 
```

Comment:
```suggestion
Testing with Bazel is still supported but not recommended unless you have prior experience 
with Bazel, and would like to use it for specific capabilities (Remote execution, etc).
This is because pytest offers many more options to run your test suite and has
better error reports, timings reports, open-source plugins and documentation online 
for Python testing. 
```